### PR TITLE
Sanitized Text in user-page.html, Added Classic editor in user-page-loader, and changed to HTML text in feed.html

### PR DESCRIPTION
**Pull request by adelwu-zz**
Screen Shot 2019-06-08 at 9 06 19 PM
In the ClassicEditor, I allowed for Header, bold, italics, link, bulleted/numbered list.

### CODE REVIEW

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ function fetchMessages() {
   const url = '/messages?user=' + parameterUsername;
   fetch(url)
       .then((response) => {
-        return response.json();
+        return response.json();
```
Comment (dhgarrette) [inline src/main/webapp/js/user-page-loader.js]: Extra space at the end of this line too.

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -97,6 +97,9 @@ function buildUI() {
   showMessageFormIfViewingSelf();
   fetchMessages();
   fetchAboutMe();
+  const config = {removePlugins: [ 'ImageUpload' , 'blockQuote'],
+                  toolbar: [ 'heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList' ]};
+  ClassicEditor.create(document.getElementById('message-input'), config );
```
Comment (dhgarrette) [inline src/main/webapp/js/user-page-loader.js]: There shouldn't be a space between `config` and `)`.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 3]
```diff
@@ -61,7 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Gson gson = new Gson();
     String json = gson.toJson(messages);
 
-    response.getWriter().println(json);
+    response.getWriter().println(json);
```
Comment (dhgarrette) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Looks like a random space got added to the end of this line.

**Review by dhgarrette — CHANGES_REQUESTED** [review]
This is so cool!  Thanks, Adel!  I just have a couple of minor comments, and also it looks like the branch needs to be synced, which you can do with `git pull origin master`.

**Comment by adelwu-zz** [comment]
Oh shoot, seems like I have a syntax error in MessageServlet somewhere in lines 87-93; can you spot it? It says there's illegal start of expressions/missing ;.

**Comment by dhgarrette** [comment]
It looks like there are still merge markers in the file:

```
<<<<<<< HEAD
    String text = Jsoup.clean(request.getParameter("text"), Whitelist.relaxed());
=======
    String text = request.getParameter("text");
>>>>>>> 9901f23991fc7478f1c1191c72aa0b627ed6f72e
```

You'll need to edit it down to just:

```
String text = request.getParameter("text");
```

**Comment by dhgarrette** [comment]
But it looks like Travis CI is working!

**Comment by adelwu-zz** [comment]
I thought I fixed the merge markers in my commit called "merge issue"? In my code I fixed it but now it won't build when I try to run the server.

**Comment by adelwu-zz** [comment]
Screen Shot 2019-06-08 at 11 46 22 PM

**Comment by dhgarrette** [comment]
So it's saying `cannot find symbol` for `Whitelist` and `Jsoup` because the file isn't importing them.  So you can add:

```
import org.jsoup.Jsoup;
import org.jsoup.safety.Whitelist;
```

to the top of that file.

**Comment by adelwu-zz** [comment]
Gotcha! Should work now :)

**Comment by dhgarrette** [comment]
Fantastic!  And the Travis CI checks are passing :-)

**Review by dhgarrette — APPROVED** [review]

**Comment by dhgarrette** [comment]
When you're ready to merge this, you can use the "Squash and merge" functionality to eliminate the intermediate commits so that the logs don't have commits for those small fixes.  
image

**Pull request merged**